src/index.html

```diff
@@ -21,9 +21,9 @@
 
 <section >
     <div class="container">
-        <h2>I'm Daniel Glad to see you</h2>
+        <h1>I'm Daniel Glad to see you</h1>
          <p >I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p> 
-         <p>LETS CONNECT</p> 
+         <h2>LET'S CONNECT</h2> 
```

Comment:
- [ ] Great job updating the elements as suggested by the previous reviewer. I noticed the styling changed completely after updating the elements. Kindly fix the styling to match the Figma template. You are expected to stick to design template as much as possible. This includes fonts, font sizes, color, background, margins and paddings.
### Your design
![image](https://user-images.githubusercontent.com/94551484/235264233-028eb10e-e12a-43cc-880e-23cf0bed4943.png)
### FIgma
![image](https://user-images.githubusercontent.com/94551484/235264271-5eca282d-3b33-4cf1-94cd-991c4870af76.png)